content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -211,11 +211,19 @@ export KUBECONFIG=/etc/kubernetes/admin.conf
 ```
 
 {{< warning >}}
-Kubeadm signs the certificate in the `admin.conf` to have `Subject: O = system:masters, CN = kubernetes-admin`.
-`system:masters` is a break-glass, super user group that bypasses the authorization layer (e.g. RBAC).
-Do not share the `admin.conf` file with anyone and instead grant users custom permissions by generating
-them a kubeconfig file using the `kubeadm kubeconfig user` command. For more details see
-[Generating kubeconfig files for additional users](/docs/tasks/administer-cluster/kubeadm/kubeadm-certs#kubeconfig-additional-users).
+The kubeconfig file `admin.conf` that `kubeadm init` generates contains a certificate with
+`Subject: O = kubeadm:cluster-admins, CN = kubernetes-admin`. The group `kubeadm:cluster-admins`
+is bound to the built-in `cluster-admin` ClusterRole.
+Do not share the `admin.conf` file with anyone.
+
+`kubeadm init` generates another kubeconfig file `super-admin.conf` that contains a certificate with
+`Subject: O = system:masters, CN = kubernetes-super-admin`.
+`system:masters` is a break-glass, super user group that bypasses the authorization layer (for example RBAC).
+Do not share the `super-admin.conf` file with anyone. It is recommended to move the file to a safe location.
```

Comment:
When might you want that file? Is it ever safer to delete it completely?

Reply:
no, i don't think deleting it is a good idea. having a break glass certificate has it's uses and that's why system:masters exists.

Reply:
OK. I have a personal view that it's safer to have the CA there but not to have a signed super-admin certificate stored anywhere (especially one that lasts more than 24 hours). However, as a docs approver, I'm happy for SIG Security and SIG Cluster Lifecycle to own the opinions here.

Reply:
as discussed on the ticket, the main problem is that users with access to hosts were sharing the system:masters credential with other users. this requires CA rotation if that credential is leaked. if a CA key leaks, CA rotation is also required, obviosly.

the request to add the feature came from a sig security member - `@raesene`
https://github.com/kubernetes/kubeadm/issues/2414